azerothcore/azerothcore-wotlk
# fix(DB): adds location information to game objects 2/17

## Changes Proposed:
- Adds map, zone and area id to game objects
- This is a continuation of #7664 
## Issues Addressed:


## SOURCE:


## Tests Performed:

- Verified that the correct amount of rows was changed — 5000 rows
- Checked every 100'th game object location — should be fine as the data was gathered automatically.
- Spot checked some data points in newly inserted data, to make sure that it was inserted correctly

## How to Test the Changes:


1. Run SQL query
2. Should change 5000 rows
3. Check some data points. No need to check all. (If you want to go ahead, but it would take 2h+)
4. Make certain that each map, zone, and area ID is in the correct column in DB

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
